src/main/scala/firrtl/Utils.scala

```diff
@@ -188,6 +188,17 @@ object Utils extends LazyLogging {
     case sx => sx
   }
 
+  /** Returns true if PrimOp is a Not, false otherwise */
+  def isNot(op: PrimOp): Boolean = op match {
+    case Not => true
+    case _ => false
+  }
```

Comment:
I would delete the `PrimOp` version because one could and should just write `op == Not`, the `Expression` version below makes sense though.

Reply:
The only reason `isCast` applies to PrimOps is that there are several of them. I could solve it by introducing a trait but I decided against it.